### DISCUSSION TITLE
v1.0.1

### DISCUSSION THREAD

**Discussion by Mrs4s** [Mrs4s/go-cqhttp#2083] [General]
## Changelog
本版本主要是修复错误读写内存导致的随机崩溃问题, 遇到该问题可尝试更新.

## 新增
0b90074 为默认的 `HTTPClient` 添加了超时设置.

## 修复
54995fc 修复一处可能会导致随机崩溃的问题.
42606a8 修复访问 `http2` 资源的时候可能会导致程序崩溃的问题.

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.0.1'>v1.0.1</a>.</em>

**Comment by Wang-Jian-sir** [comment]
登录失败: 你当前使用的QQ版本过低，请前往QQ官网im.qq.com下载最新版QQ后重试。 Code: 45
 你的账号涉嫌违规被限制在非常用设备登录, 请在手 机QQ登录并根据提示完成认证，我小号会有这个问题，换个大号倒是可以正常运行。

**Comment by yuweuss** [comment]
怎么更新啊！大佬？是不是不更新就不能用了？

**Comment by LevenYi** [comment]
没有linux-amd64了吗

**Comment by xiaokun5252** [comment]
不懂就问：手动更新需要添加 -update-protocol 参数启动、这个怎么添加。

**Reply by Wang-Jian-sir** [reply]
在生成的.bat文档内，start cmd /K "go-cqhttp.exe" -faststart -update-protocol，这个是跳过开始5秒并自动更新，https://docs.go-cqhttp.org/guide/quick_start.html，这里有帮助文档

**Reply by xiaokun5252** [reply]
登录失败: 登录失败，建议升级最新版本后重试，或通过问题反馈与我们联系。 Code: 45
你的账号涉嫌违规被限制在非常用设备登录, 请在手机QQ登录并根据提示完成认证
或使用 -update-protocol 升级到最新协议后重试

**Reply by yyx2684264** [reply]
> 登录失败: 登录失败，建议升级最新版本后重试，或通过问题反馈与我们联系。 Code: 45 你的账号涉嫌违规被限制在非常用设备登录, 请在手机QQ登录并根据提示完成认证 或使用 -update-protocol 升级到最新协议后重试

最近好多机器人都出现了这个问题，包括但不限于icqq,ocqq，感觉是腾讯在搞事情

**Reply by xiexiaogongyang** [reply]
我也有这个情况，zc.qq.com再注册一个账号吧

**Comment by Mjfare** [comment]
我最近部署的cq-http机器人在QQ上，前几个月一直都没有问题，但最近的机器人一天发言二三十次之后就被风控，或者可能发言频繁一点就被风控。提示群消息发送失败: 账号可能被风控，然后更换了好几个QQ号都是这样的现象，哪怕是我自己的大号天天登录的QQ号都是被风控。但是当天被风控隔天早上就解除风控，而且只是机器人发送失败，自己本人的发言不受影响。请问这个应当怎么解决呀？我自己写的代码会导致这一情况吗？感谢解答！

**Comment by 746787419** [comment]
ios客户端收到qq语音和图片（cq码格式发出的）有很大概率出现空语音和不显示图片的空消息，但是安卓和pc端都可以正常显示出来，感觉很奇怪。
